packages/semver2/semver2.1.2.0/opam

```diff
@@ -0,0 +1,20 @@
+opam-version: "2.0"
+maintainer: "info@dividat.com"
+authors: ["Dividat AG"]
+synopsis: "Semantic version handling for OCaml"
+license: "MIT"
+homepage: "https://github.com/dividat/ocaml-semver"
+bug-reports: "https://github.com/dividat/ocaml-semver/issues"
+depends: [
+  "ocaml" {>= "4.04.0"}
+  "dune" {>= "1.2.0"}
+  "angstrom" {>= "0.14.0"}
+  "ounit" {with-test & >= "1.0.2"}
+  "yojson" {with-test & >= "1.4.1"}
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {pinned}
```

Comment:
```suggestion
  ["dune" "subst"] {dev}
```

Reply:
Please make this change also upstream (no need to re-tag or re-release)

Reply:
Is there some context for this?

`dune`'s docs still prescribe `pinned`: https://github.com/ocaml/dune/blob/2.9/doc/opam.rst#invocation-from-opam

Reply:
Nevermind, just saw your PR in the dune project. 👍